src/plugins/notebook/components/NotebookEntry.vue

```diff
@@ -14,6 +14,9 @@
                  class="c-ne__text"
                  :class="{'c-ne__input' : !readOnly }"
                  :contenteditable="!readOnly"
+                 tabindex="0"
+                 @keydown.enter.exact.prevent
```

Comment:
Not saying this is wrong but do we need this?

Reply:
yes, `contenteditable` has default behavior of newline on enter, need to prevent that.

Reply:
ahhh thank you